### DISCUSSION TITLE
"Failed to parse project xml"

### DISCUSSION THREAD

**Discussion by taginu** [vkbo/novelWriter#2004] [Help] [answered]
I have 1 pc with Debian 12.5 and earlier a laptop also with Debian 12.5. I have my manuscript files on a external SSD drive but I have NovelWriter installed localy on both computers. I am not moving the manuscript files into either of my two computers. Instead I plug the ssd in to any of the computers and work directly with the files. So the files never leave the external SSD. It has worked well for a while. I recently bought a new laptop and due to lack of realtek network card driver in Debian I installed Fedora 40 on the new laptop and that works fine with laptop network hardware. It is the same user-name and group-name on the both computers. 

When I try to access the manuscript files on the SSD with NovelWriter I get a little error window in NovelWriter which says "Failed to parse project xml". I can't figure out a solution to that. Do you have any idea?

**Comment by vkbo** [comment]
You probably have a corrupt `nwProject.nwx` file in your project folder. You can open it in a plain text editor and check. It should be a valid XML file.

This is a generic error that can mean a few things. If you don't mind starting novelWriter from the command line, then run:

```
novelwriter --debug /path/to/project/directory
```

Then post the last few lines of log output around the error. It should print more detailed information there.

**Reply by taginu** [reply]
On my Debian-machine i have installed NovelWriter via .deb installation file. On the laptop with Fedora I downloaded .AppImage version.

When I open the project in the stationary Debian based (.deb) pc, it works fine. The manuscript project folder seems to be OK.
When I try to run --debug on the Fedora-machine I write following command line:
"/home/ba/apps/novelwriter-2.5.1.AppImage --debug /run/media/ba/Boken/Gå\ vilse" it says "The project is already open..." and I can't remove the .lock file.
See the attached log file:

[log.txt](https://github.com/user-attachments/files/16592162/log.txt)

I

**Reply by vkbo** [reply]
This is the relevant part in the log:

```
[2024-08-12 23:14:04,918]          guimain.py:339   INFO      Command line path: /run/media/ba/Boken/Gå vilse
[2024-08-12 23:14:04,918]          project.py:252   INFO      Opening project: /run/media/ba/Boken/Gå vilse
[2024-08-12 23:14:04,919]          storage.py:360   ERROR     Failed to read project lockfile
[2024-08-12 23:14:04,919]          storage.py:361   ERROR     OSError: [Errno 5] Input/output error: '/run/media/ba/Boken/Gå vilse/nwProject.lock'
[2024-08-12 23:14:04,919]          storage.py:212   ERROR     Project is locked, so not opening
```

It looks like the disk cannot be read.

**Reply by taginu** [reply, marked as answer]
The SSD probably requires more power than the laptop's USB connector can provide, which I will investigate further tomorrow. Many thanks for your guidance.
Ett stort tack!
/Magnus
